typescript/processors/readTypeScriptModules.js

```diff
@@ -140,9 +156,65 @@ module.exports = function readTypeScriptModules(tsParser, modules, getFileInfo,
           }
         });
       });
+
+      /** Resolves the inherited docs and stores them in the export doc. */
+      function resolveInheritedDocs(exportSymbol, exportDoc) {
+        var inheritedSymbols = resolveInheritedSymbols(exportSymbol, typeChecker);
+        var inheritedDocs = inheritedSymbols.map(function(symbol) {
+          var relatedDoc = docs.find(function(doc) { return doc.exportSymbol === symbol; });
```

Comment:
This could be super slow, given that we are doing a search of o(n) on every single inheritance symbol.